charts/velocity-operator/values.yaml

```diff
@@ -42,3 +42,50 @@ metricsService:
   type: ClusterIP
 secrets:
   authToken: replace-me
+
+aws:
+  region: &aws-region us-east-1
+  creds:
+    accessKey:
+    secretKey:
+
+    k8sSecretName: &k8s-secret-name aws-creds
+    k8sSecretKeyName: &k8s-secret-key-name credentials
+    profile: &aws-profile default
+
```

Comment:
It's not missing its deliberate.

It's a bit hard to explain the behaviour with tags vs conditions ([see here](https://helm.sh/docs/chart_best_practices/dependencies/#conditions-and-tags))

but if I include it, I have to include a value `true/false`. If true it will always override the tags. If false, then the default behaviour of "enable all cloud resources" will not work

Reply:
interesting. cool thanks